### PR TITLE
Announce Selenium 4

### PR DESCRIPTION
**Pull request by shs96c**
### Description
Announce Selenium 4

### Motivation and Context
We want the world to know we've finally shipped this thing

### Types of changes
- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/blog/2021/announcing-selenium-4.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+marked as internal to Selenium, or that was deprecated, you might
+experience some hiccups. Please [check our documentation][docs] for
+advice on how to deal with each of the common problems we’re aware of.
+
+But there’s more to Selenium 4 than just being a stable release of
+what was there before! It brings a whole host of new and exciting
+features that we hope will make your tests more fun to write, and more
+stable when you run them. Let’s take a look at some of them!
+
+We’ve introduced “[relative locators]”. These allow you to specify
```
Comment (AutomatedTester) [inline website_and_docs/content/blog/2021/announcing-selenium-4.md]: does this need to be 
```suggestion
We’ve introduced “[relative locators][relative locators]”. These allow you to specify
```

File: website_and_docs/content/blog/2021/announcing-selenium-4.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+[Selenium-Level sponsors][sponsors] for getting this release ready for
+our users.
+
+**We hope you enjoy Selenium 4, and we can’t wait to see what you do
+with it!**
+
+[auth]: /documentation/webdriver/bidi_apis/#register-basic-auth
+[browserstack]: https://www.browserstack.com/
+[chromium]: https://www.chromium.org/Home
+[docker]: https://hub.docker.com/u/selenium
```
Comment (diemol) [inline website_and_docs/content/blog/2021/announcing-selenium-4.md:104]:
```suggestion
[docker]: https://github.com/seleniumhq/docker-selenium/
```

File: website_and_docs/content/blog/2021/announcing-selenium-4.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+[docs]: http://www.google.com
+[firefox]: https://www.mozilla.org/en-GB/firefox/new/
+[github]: https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.0.0
+[grid]: /documentation/grid/
+[js errors]: https://www.selenium.dev/documentation/webdriver/bidi_apis/#listen-to-js-exceptions
```
Comment (diemol) [inline website_and_docs/content/blog/2021/announcing-selenium-4.md]:
```suggestion
[js errors]: /documentation/webdriver/bidi_apis/#listen-to-js-exceptions
```

File: website_and_docs/content/blog/2021/announcing-selenium-4.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+[mutation]: https://www.selenium.dev/documentation/webdriver/bidi_apis/#mutation-observation
```
Comment (diemol) [inline website_and_docs/content/blog/2021/announcing-selenium-4.md]:
```suggestion
[mutation]: /documentation/webdriver/bidi_apis/#mutation-observation
```

File: website_and_docs/content/blog/2021/announcing-selenium-4.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+our users.
+
+**We hope you enjoy Selenium 4, and we can’t wait to see what you do
+with it!**
+
+[auth]: /documentation/webdriver/bidi_apis/#register-basic-auth
+[browserstack]: https://www.browserstack.com/
+[chromium]: https://www.chromium.org/Home
+[docker]: https://hub.docker.com/u/selenium
+[docs]: http://www.google.com
```
Comment (diemol) [inline website_and_docs/content/blog/2021/announcing-selenium-4.md]:
```suggestion
[docs]: /documentation/getting_started/how_to_upgrade_to_selenium_4/
```

**Review by diemol — CHANGES_REQUESTED** [review]

**Review by diemol — APPROVED** [review]

**Pull request merged**